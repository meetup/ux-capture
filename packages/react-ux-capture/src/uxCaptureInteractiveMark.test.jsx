import React from 'react';
import UXCaptureInteractiveMark from './UXCaptureInteractiveMark';

const MOCK_PROPS = {
	mark: 'bar',
};

const renderComponent = (props = MOCK_PROPS) =>
	<UXCaptureInteractiveMark {...props}>
		<b>Bold Child</b>
	</UXCaptureInteractiveMark>;

describe('UXCaptureInteractiveMark', () => {
	it('renders children markup', () => {
		expect(renderComponent()).toMatchSnapshot();
	});
});
