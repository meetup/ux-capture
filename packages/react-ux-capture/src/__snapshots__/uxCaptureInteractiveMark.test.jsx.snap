// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UXCaptureInteractiveMark renders children markup 1`] = `
<UXCaptureInteractiveMark
  mark="bar"
>
  <b>
    Bold Child
  </b>
</UXCaptureInteractiveMark>
`;
